src/plugins/discover/public/application/components/hits_counter/hits_counter.test.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { ReactWrapper } from 'enzyme';
+import { HitsCounter, HitsCounterProps } from './hits_counter';
+// @ts-ignore
+import { findTestSubject } from '@elastic/eui/lib/test';
+
+describe('hits counter', function() {
+  let props: HitsCounterProps;
+  let component: ReactWrapper<HitsCounterProps>;
+
+  beforeAll(() => {
+    props = {
+      onResetQuery: jest.fn(),
+      showResetButton: true,
+      hits: 2,
+    };
+  });
+
+  it('HitsCounter renders a button by providing the showResetButton property', () => {
+    component = mountWithIntl(<HitsCounter {...props} />);
+    expect(component).toMatchSnapshot();
```

Comment:
I'd suggest, since a snapshot tests in this case adds a lot of LOC and it's likely that we'd have to regenerate it, if some of the underlying components changed, to use `findTestSubject` for `resetSavedSearch`

Reply:
Sure you are right!